streamz/core.py

```diff
@@ -351,7 +351,8 @@ def update_frequencies(last, x):
 
         return self.scan(update_frequencies, start={}, **kwargs)
 
-    def visualize(self, filename='mystream.png', source_node=False, **kwargs):
+    def visualize(self, filename='mystream.png', source_node=False,
+                  rankdir='LR', **kwargs):
```

Comment:
@CJ-Wright , this might affect you.  Any objections?

Reply:
What was the reason for this? I'm generally ok with this, since I can override it.

Reply:
I generally prefer left-to-right as a default over top-to-bottom.  I can go either way though.

Reply:
I guess we should default to something. I usually prefer having the stream direction and the names orthogonal so that we fill the page more, but I don't mind putting in the kwarg.

Reply:
OK, removing.  Given the difference in opinion we'll stick to graphviz' default

Reply:
Thank you. I'll make a point of explicitly putting top-bottom in my code just in case we do make the shift.